portal.go

```diff
@@ -1012,6 +1012,12 @@ func (portal *Portal) HandleMatrixMessage(evt *event.Event) {
 	if portal.bridge.IM.Capabilities().RichLinks {
 		imessageRichLink = portal.convertURLPreviewToIMessage(evt)
 	}
+	var metadata *interface{}
+	if portal.bridge.IM.Capabilities().Metadata {
```

Comment:
These capability checks seem a bit pointless, it can just pass it unconditionally and nothing will break